crt_portal/cts_forms/migrations/0085_remove-assignee_20200716.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.2.13 on 2020-07-16 15:26
+
+from django.db import migrations
+from datetime import datetime
+import pytz
+from actstream import action
+from django.contrib.auth import get_user_model
+from actstream import registry
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cts_forms', '0084_form_template_standardization'),
+    ]
+
+    def remove_assignee(apps, schema_editor):
+        # get all reports
+        reports = apps.get_model('cts_forms', 'Report')
+        # user = apps.get_model('cts_forms', 'User')
+        User = get_user_model()
+        userlist = User.objects.all().order_by('id')
+        # // Assuming that first user is the root user or superuser. If there are on users, create a dummy one
+        if len(userlist) > 0:
+            superuser = userlist[0]
+        else:
+            superuser = User.objects.create_user('migration_user')
```

Comment:
This isn't a bad idea, but my recommendation would be to skip the migration if we don't have an user. I think this keeps things somewhat simpler.